### PR TITLE
deprecated ts-nkeys now that nkeys.js provides support for all of supported JavaScript runtimes

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,3 @@
...
-
-## Supported Node Versions
-
-Our support policy for Nodejs versions follows [Nodejs release support]( https://github.com/nodejs/Release).
-We will support and build node-nats on even-numbered Nodejs versions that are current or in LTS.
-
-## License
-
-Unless otherwise noted, the NATS source files are distributed under the Apache Version 2.0 license found in the LICENSE file.
+ts-nkeys is not part of [nkeys.js](github.com/nats-io/nkeys.js) which provides support for all of our supported JavaScript environments (Node.js, Browser, Deno). For more information please visit [github.com/nats-io/nkeys.js](github.com/nats-io/nkeys.js).
```
Comment (kozlovic) [inline README.md:3]: `not part` or `now part`?

Comment (aricart) [reply README.md]: Thanks for the catch

**Review by kozlovic — APPROVED** [review]
LGTM

**Comment by jimmywarting** [comment]
Maybe you should also archive this repo and depricate the npm package too?

**Pull request merged**